docs: add example for loading multiple images with kind load docker-i…

AnjaliMishra1st opened the pull request:
This PR improves the Quick Start guide by adding an example showing how to
load multiple images at once with `kind load docker-image.

Example 
kind load docker-image my-app:latest my-db:latest my-cache:latest --name test-cluster

AnjaliMishra1st commented:
Hi @BenTheElder @stmcginnis 👋,

Just following up on this PR. Since this is part of my GSSoC contribution, I wanted to check if you could please take a quick look when you get a chance 🙏.

The PR adds a simple docs example for loading multiple images with kind load docker-image. It’s small (size/XS) and ready for review. Looking forward to your feedback! 😊

Thanks a lot for your time and support.

stmcginnis reviewed (commented):
Thanks for working on improving the docs! This seems like a decent idea to point out you can run multiple imports at once.

We already have a section describing the `load` command earlier in the doc. I think things would flow better if this was just a sentence or two there pointing out that it is an option.

Thanks!

AnjaliMishra1st commented:
Hi @stmcginnis, thanks for pointing that out 🙌
I’ve fixed the missing backticks and updated the code block formatting in quick-start.md. Please have another look when you get a chance.

stmcginnis commented on site/content/docs/user/quick-start.md:
```diff
@@ -494,3 +494,10 @@ kind, the Kubernetes cluster itself, etc.
 [access multiple clusters]: https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/
 [release notes]: https://github.com/kubernetes-sigs/kind/releases
+
+### Loading multiple images into your cluster
```
Sorry, I think you misunderstood or I wasn't as specific as I should have been. There is [an existing section](https://github.com/kubernetes-sigs/kind/blob/main/site/content/docs/user/quick-start.md?plain=1#L219) where this note should be added to. It should not be another new section at the end of the document.

BenTheElder commented on site/content/docs/user/quick-start.md:
```diff
@@ -494,3 +494,18 @@ kind, the Kubernetes cluster itself, etc.
 [customize control plane with kubeadm]: https://kubernetes.io/docs/setup/independent/control-plane-flags/
 [access multiple clusters]: https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/
 [release notes]: https://github.com/kubernetes-sigs/kind/releases
```
nit: we should prefer to keep this set of links at the end of the document and insert any new sections before them

stmcginnis commented on site/content/docs/user/quick-start.md:
```diff
@@ -218,13 +218,18 @@ context name `kind` and delete that cluster.
 
-> **Note**: If using a named cluster you will need to specify the name of the
-> cluster you wish to load the images into:
-> `kind load docker-image my-custom-image-0 my-custom-image-1 --name kind-2`
```
Sorry, please keep the need for passing the cluster name separate as for a very large number of users, they only start the default kind cluster and don't need to worry about additional parameters. That will also keep the examples above more concise.

stmcginnis commented on site/content/docs/user/quick-start.md:
```diff
@@ -218,13 +218,13 @@ context name `kind` and delete that cluster.
 ## Loading an Image Into Your Cluster
 
-Docker images can be loaded into your cluster nodes with:
+You can also load more than one image at once by passing multiple image names:
```
This is now missing the single image example. Looks like it was added below. That section should move up and replace this section's content here.

stmcginnis replied:
With the correction for not passing the explicit cluster name and just keeping that detail in the existing note.

stmcginnis commented on site/content/docs/user/quick-start.md:
```diff
@@ -157,7 +157,7 @@ wait for 30 seconds, do `--wait 30s`, for 5 minutes do `--wait 5m`, etc.
 More usage can be discovered with `kind create cluster --help`.
 
-The kind can auto-detect the [docker], [podman], or [nerdctl] installed and choose the available one. If you want to turn off the auto-detect, use the environment variable `KIND_EXPERIMENTAL_PROVIDER=docker`, `KIND_EXPERIMENTAL_PROVIDER=podman` or `KIND_EXPERIMENTAL_PROVIDER=nerdctl` to
+kind can auto-detect the [docker], [podman], or [nerdctl] installed and choose the available one. If you want to turn off the auto-detect, use the environment variable `KIND_EXPERIMENTAL_PROVIDER=docker`, `KIND_EXPERIMENTAL_PROVIDER=podman` or `KIND_EXPERIMENTAL_PROVIDER=nerdctl` to
```
nit: I think in this case we should uppercase `kind` since it is a little odd as the start of the sentence.

stmcginnis reviewed (commented):
Almost there.

AnjaliMishra1st commented:
Thanks for catching that @stmcginnis!
I removed the extra blank lines at the bottom. Please take another look when you get a chance. 🙏

stmcginnis commented:
Yes, there is still the one pending format issue to update. Looks like once you do that it should be ready to go.

AnjaliMishra1st commented:
Hi @stmcginnis @BenTheElder,  

I’ve updated the "Loading an Image Into Your Cluster" section:  
- Split single and multiple image examples into separate fenced code blocks.  
- Removed stray `bash` and extra backticks.  
  
Please take another look when you get a chance. 🙏

aojea commented on site/content/docs/user/quick-start.md:
```diff
@@ -220,11 +220,15 @@ context name `kind` and delete that cluster.
+
+You can also load more than one image at once by passing multiple image names:
 
-> **Note**: If using a named cluster you will need to specify the name of the
```
why is this removed?

AnjaliMishra1st commented:
Hi @stmcginnis @BenTheElder @aojea,
I’ve kept the PR as two commits to ensure the formatting and examples remain intact. All changes are related and ready for merge. Please let me know if anything else is needed. 🙂

AnjaliMishra1st commented:
Thanks for pointing this out @stmcginnis 🙏. I’ve been trying to squash into a single commit, but each time I do, the formatting shifts and the section gets duplicated at the bottom.

AnjaliMishra1st commented:
Hi @stmcginnis 👋,
Thanks again for your review and patience 🙏. I’ve been trying to squash into a single commit, but every time I do, the formatting shifts and the section gets duplicated at the bottom.

Could you please guide me on the correct way to squash in this case? I think I might be missing a step while rebasing/resetting, and I want to make sure I follow the project’s best practices.

Once I know the right approach, I’ll apply it and update the PR right away .

stmcginnis commented:
Now that you have a single commit it may be easier. From where you are now, edit the file and make the necessary changes to get it how it needs to be. If you are using something like VSCode, you can press Ctrl-Shift-V (or Cmd-Shift-V on Mac) to view a rendered version of the file where you can make sure it looks like you expect it to.

Then rather than creating another commit and squashing, just do:

```
git add .
git commit --amend
```

That will allow you to edit the existing commit and make any changes to the commit message, if you want or need to. Then close out of what ever editor your system is configured to use for editing commit messages and git will have just updated your single, sole commit. That's what we're ultimately going for. Then `git push -f origin docs-kind-load-images` (or whatever your remote is called, if not `origin`) to update this PR.

AnjaliMishra1st commented:
Hi @stmcginnis 👋,  
I’ve updated the `quick-start.md` file — fixed the formatting (added the missing blank lines after headings and around code blocks) and removed the duplicate section.  

Thanks for catching that 🙏. Please let me know if anything else needs tweaking!

stmcginnis commented:
Thanks!

/lgtm

aojea commented:
/approve

Thanks
